app/controllers/account/admin/users_controller.rb

```diff
@@ -0,0 +1,36 @@
+# Encoding: utf-8
+# frozen_string_literal: true
+module Account
+  module Admin
+    class UsersController < ApplicationController
+      def index
+        @users = User.page(params[:page]).per(20)
+        authorize @users
```

Comment:
This one is already in progress - https://github.com/Kotany-IF070-WebUI-RoR/NewVasjuky/issues/46